packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {helpers, ShopperLogin, ShopperLoginTypes} from 'commerce-sdk-isomorphic'
+import jwtDecode from 'jwt-decode'
+import {ApiClientConfigParams} from '../hooks/types'
+import {BaseStorage, LocalStorage, CookieStorage} from './storage'
+
+type Helpers = typeof helpers
+interface AuthConfig {
+    shortCode: string
+    organizationId: string
+    clientId: string
+    siteId: string
+    redirectURI: string
+    proxy: string
+}
```

Comment:
Minor: We can extend this from another interface like this:

```suggestion
interface AuthConfig extends ApiClientConfigParams {
    redirectURI: string
    proxy: string
}
```